File: android/src/test/java/org/ligi/satoshiproof/TheAddressGenerator.java
package org.ligi.satoshiproof;

import org.junit.Test;
import static org.assertj.core.api.Assertions.assertThat;
import static org.ligi.satoshiproof.util.AddressGenerator.dataToAddressString;

public class TheAddressGenerator {

    @Test
    public void shouldGenerateCorrectAddresses() {
        assertThat(dataToAddressString("probe".getBytes())).isEqualTo("1KXgQMLN5ceej3ViannhJudCjJsFWrWXvs");

        assertThat(dataToAddressString("foo".getBytes())).isEqualTo("1MaybZp8GRkAHmpAyWkSQEwAnohFxPBoGY");
    }
}

File: android/src/test/kotlin/org/ligi/satoshiproof/TheAddressGenerator.kt
package org.ligi.satoshiproof

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test
import org.ligi.satoshiproof.util.AddressGenerator.dataToAddressString

class TheAddressGenerator {

    @Test
    fun shouldGenerateCorrectAddresses() {
        assertThat(dataToAddressString("probe".toByteArray())).isEqualTo("1KXgQMLN5ceej3ViannhJudCjJsFWrWXvs")

        assertThat(dataToAddressString("foo".toByteArray())).isEqualTo("1MaybZp8GRkAHmpAyWkSQEwAnohFxPBoGY")
    }
}